src/components/Header/Header.jsx

```diff
@@ -1,11 +1,13 @@
+import { useEntries } from '../../context/PlannerContext';
 import styles from './Header.css';
 
 export default function Header() {
+  const { entries } = useEntries([]);
   return (
     <header className={styles.header}>
       <h2>My Planner</h2>
       {/* TODO: Replace hard-coded count with the actual number of entries */}
-      <span>1 Item Left</span>
+      <span>{entries.length}Item Left</span>
     </header>
   );
```

Comment:
We can also use a ternary here to check if we should display 'item left' or 'items left' by checking the length of entries.